src/sql/platform/errorMessage/common/errorMessageService.ts

```diff
@@ -10,5 +10,5 @@ import { createDecorator } from 'vs/platform/instantiation/common/instantiation'
 export const IErrorMessageService = createDecorator<IErrorMessageService>('errorMessageService');
 export interface IErrorMessageService {
 	_serviceBrand: undefined;
-	showDialog(severity: Severity, headerTitle: string, message: string, messageDetails?: string, actions?: IAction[]): void;
+	showDialog(severity: Severity, headerTitle: string, message: string, messageDetails?: string, actions?: IAction[], instructionText?: string): void;
```

Comment:
Please add doc comments to this method - you can leave the existing ones blank if you don't have a good sense of what they do right now but let's at least get one filled in for this new parameter.